ledger/ledger-api-test-tool/BUILD.bazel

```diff
@@ -108,6 +112,8 @@ da_scala_binary(
         "src/main/resources/logback.xml",
         ":PackageManagementTest",
         "//ledger/test-common:dar-files",
+    ] + [
+        "//:MVN_VERSION",
```

Comment:
This can probably be dropped now, thanks.

Reply:
Oh, i guess it is brought in by the dependency. OK